src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin: 16px 0;
+  margin-bottom: 19px;
+  margin-left: 1px;
+
+  &__block {
+    display: flex;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 1rem;
+    height: 1rem;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
The modifiers don't work, consider fixing it

Reply:
sorry, I can't figure out how to fix it